### ISSUE TITLE
blank screen on ubuntu after suspend

### ISSUE THREAD

**Issue by baermathias** [Eugeny/terminus#3339]
This is what I see under Ubuntu 20.04 after suspend:

![Screenshot from 2021-01-14 16-47-57](https://user-images.githubusercontent.com/45003352/104614318-6ed62100-5688-11eb-8f2c-2173abbba485.png)

All other apps work normal

**Comment by kappelmann** [comment]
Facing the same problem

**Comment by aqueos** [comment]
yes this is happening in 20.04 to me as well. Each suspend break the tabby. I need to reset the theme to another one, refresh each windows then go back to the original theme and refresh all tab again. 

regards,Ghislain.

**Comment by kjteske** [comment]
Same for me on Ubuntu 20.04. Happy to collect logs / help troubleshoot if needed.

**Comment by aqueos** [comment]
the problem still exist in 

Ubuntu 22.04.1 LTS
ii  tabby-terminal 1.0.183      amd64        

each time i suspend the PC the tabby window came back all black again.

best regards,
Ghislain.

**Comment by konstantin83** [comment]
Same here. Ubuntu 
![tabby](https://github.com/Eugeny/tabby/assets/13274062/462d1ab5-c941-42aa-a800-1662147dd8cf)
![linux](https://github.com/Eugeny/tabby/assets/13274062/734c7326-966d-4929-956f-8bf8ea9b3d81)

**Comment by csantosbh** [comment]
Also facing this issue on 22.04. Are there any workarounds?

**Comment by mknap** [comment]
Same on 24.10. Wayland. Nvidia. Anything I can do to help track this down? 

I cannot even seem to refresh the old terminals. I have to close them and start new ones.
